src/Microsoft.AspNetCore.Localization/CookieRequestCultureProvider.cs

```diff
@@ -118,5 +145,8 @@ public static ProviderCultureResult ParseCookieValue(string value)
 
             return new ProviderCultureResult(cultureName, uiCultureName);
         }
+
+        private static bool IsValidCulture(string cultureName) =>
+            CultureInfo.GetCultures(CultureTypes.SpecificCultures).Any(c => c.Name.Equals(cultureName));
```

Comment:
@Eilon and @pranavkm are we happy with these checks now?

Reply:
Nope. We have a separate discussion about this here: https://github.com/aspnet/Localization/pull/385#discussion_r134241089.

Reply:
Please have a look to my comment [here](https://github.com/aspnet/Localization/pull/385#discussion_r134309983)